src/SwedbankPay.Sdk/CurrencyCode.cs

```diff
@@ -13,30 +13,43 @@
 using System.Globalization;
 using System.Linq;
 
-using Newtonsoft.Json;
-
 namespace SwedbankPay.Sdk
 {
+    /// <summary>
+    /// The currency code used for the payment
+    /// </summary>
     public class CurrencyCode
```

Comment:
Can this class be renamed to just `Currency` perhaps?

```suggestion
    public class Currency
```